pulsar-client-cpp/tests/standalone.conf

```diff
@@ -0,0 +1,254 @@
+#
```

Comment:
Instead of using a special config file, we could change the tests to point to port 6650